### PR TITLE
Enable rdfa mode for rbs and some tweaks

### PR DESCRIPTION
**Pull request by abeforgit**
Mainly adding the snippet insert feature and the css import

### Setup

just npm i and run, the plugins are connected with github link

Note: to connect to the futurenow RB backend when running locally, you'll need to change the endpoints in config/environment.js
I debated adding that here but I wanna avoid frantic debugging due to forgetting this was changed.

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [ ] changelog
- [ ] npm lint
- [ ] no new deprecations

### CODE REVIEW

File: app/controllers/regulatory-statements/edit.js [hunk 1]
```diff
@@ -183,6 +186,12 @@ export default class RegulatoryStatementsRoute extends Controller {
     ];
   }
 
+  get activeNode() {
+    if (this.controller) {
+      return getActiveEditableNode(this.controller.activeEditorState);
```
Comment (piemonkey) [inline app/controllers/regulatory-statements/edit.js:191]: `getActiveEditableNode()` seems to get `undefined` whenever an ember-node is selected, which then causes an exception in the snippet plugin, leaving the editor in a partially-working state. For example, by adding/clicking on a link node.

**Review by piemonkey — COMMENTED** [review]
Creating a resource node and adding a `say:allowedSnippetList` named node property with snippet list uris from futurenow works. For some reason I'm seeing an intl warning for a translation that looks to exist in the plugins repo:
```
[ember-intl] translation: "snippet-plugin.alert.loading" on locale: "en-us, nl-be" was not found.
```

The crash noted in an inline comment is the only showstopper, though it's likely a fix in the plugins repo.

File: app/controllers/regulatory-statements/edit.js (hunk 1 above)
Comment (elpoelma) [reply app/controllers/regulatory-statements/edit.js:191]: getting the same error, will check if I can reproduce it in the plugins repo.

Comment (elpoelma) [reply app/controllers/regulatory-statements/edit.js:191]: This should be fixed now. A conditional checking if `activeNode` is defined was missing.

**Comment by elpoelma** [comment]
> Creating a resource node and adding a `say:allowedSnippetList` named node property with snippet list uris from futurenow works. For some reason I'm seeing an intl warning for a translation that looks to exist in the plugins repo:
> 
> ```
> [ember-intl] translation: "snippet-plugin.alert.loading" on locale: "en-us, nl-be" was not found.
> ```
> 
> The crash noted in an inline comment is the only showstopper, though it's likely a fix in the plugins repo.

Translation issue should also be fixed now.

**Review by elpoelma — APPROVED** [review]

**Pull request merged**